Web3/Classes/ContractABI/ABI/SolidityType.swift

```diff
@@ -158,7 +158,7 @@ public indirect enum SolidityType {
 
 public extension SolidityType.ValueType {
     
-    public var nativeType: ABIConvertible.Type? {
```

Comment:
@ypopovych @pixelmatrix Same as above. I can't see a reason to restrict access to read only fields in our API. But I may be wrong here...